server/builtin/install_consent.go

```diff
@@ -132,7 +132,7 @@ func (a *builtinApp) newInstallConsentForm(m apps.Manifest, creq apps.CallReques
 	fields = append(fields, deployTypeField)
 
 	// JWT secret
-	if deployType == apps.DeployHTTP {
+	if deployType == apps.DeployHTTP && m.SupportsDeploy(apps.DeployHTTP) && m.HTTP.UseJWT {
```

Comment:
If we have a AWS app that also support HTTP... will this show the JWT field even if we are trying to install it as a AWS app?

Reply:
AWS Apps should not be checking JWTs, they should use the API Gateway for that if deployed as HTTP.